cmake/templates/conf.py.in

```diff
@@ -77,9 +77,9 @@ for lib_dir in os.listdir(hpx_libs_dir):
         # If we are in the root of the library dir, we add all
         # subdirectories as a new breathe project
         if len(subdir) == 0:
-            for dir in dirs:
-                breathe_projects_source[dir] = (subdir_full + '/' + dir, [])
-                hpx_libs[lib_dir].append(dir)
+            for Dir in dirs:
```

Comment:
How about renaming `dir` to `directory` and `dirs` to `directories` or something similar? `snake_case` is pretty well established for variable names (and the rest of the file follows this) so it would fit significantly better.

Reply:
Alternative: `dir` to `d`. Short, but quite obvious from the context that it's a directory.

Reply:
Makes sense. Went the more descriptive route and chose `directory`.